webpack/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import URI from 'urijs';
+import { mount } from 'foremanReact/common/MountingService';
+import componentRegistry from 'foremanReact/components/componentRegistry';
+import JobInvocationContainer from './react_app/components/job_invocations';
+import './react_app/redux/reducers'
+
+componentRegistry.register(
+  { name: 'JobInvocationContainer', type: JobInvocationContainer}
+);
+
+if(window.location.href.match(/job_invocations/)) {
```

Comment:
why do this way? maybe you should use react-router like katello does? I'm worried this always require a full page reload for this code to be executed.